test/invoice_repository_test.rb

```diff
@@ -83,16 +83,16 @@ def test_it_finds_all_by_status
   end
 
   def test_finds_all_by_created_at
-    result1 = invoice_repository.find_all_by_created_at(DateTime.parse('2012-03-25 09:54:09 UTC'))
-    result2 = invoice_repository.find_all_by_created_at(DateTime.parse('2015-03-25 09:54:09 UTC'))
+    result1 = invoice_repository.find_all_by_created_at('2012-03-25 09:54:09 UTC')
+    result2 = invoice_repository.find_all_by_created_at('2015-03-25 09:54:09 UTC')
     assert_equal Array, result1.class
     assert_equal 1, result1.length
     assert_equal 0, result2.length
   end
 
   def test_finds_all_by_updated_at
-    result1 = invoice_repository.find_all_by_updated_at(DateTime.parse('2012-03-25 09:54:09 UTC'))
-    result2 = invoice_repository.find_all_by_updated_at(DateTime.parse('2015-03-25 09:54:09 UTC'))
+    result1 = invoice_repository.find_all_by_updated_at('2012-03-25 09:54:09 UTC')
```

Comment:
Line is too long. [82/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.